rest/data/distro.go

```diff
@@ -73,7 +67,7 @@ func (dc *DBDistroConnector) UpdateDistro(old, new *distro.Distro) error {
 }
 
 // CreateDistro inserts the given distro.Distro.
-func (dc *DBDistroConnector) CreateDistro(distro *distro.Distro) error {
+func CreateDistro(distro *distro.Distro) error {
```

Comment:
Similarly I think this doesn't need to be it's own function since it's only wrapping one operation and this error is already going to be checked outside of the function